src/EnergyPlus/WindowEquivalentLayer.cc

```diff
@@ -6991,61 +6991,79 @@ bool RB_LWP(CFSLAYER const &L, // RB layer
 }
 
 bool RB_SWP(EnergyPlusData &state,
-            CFSLAYER const &L,                      // RB layer
-            CFSSWP &LSWP,                           // returned: equivalent layer properties set
-            ObjexxFCL::Optional<Real64 const> THETA // incident angle, 0 <= theta <= PI/2
+            CFSLAYER const &L, // RB layer
+            CFSSWP &LSWP,      // returned: equivalent layer properties set
+            const Real64 THETA // incident angle, 0 <= theta <= PI/2
```

Comment:
This function uses the presence of the optional to determine which calculation is to be done, but the function isn't called with an `Optional` argument and the function s fairly simple, so break it into two and overload. This is the version with the argument.